portal/static/portal/sass/styles.scss

```diff
@@ -811,74 +818,122 @@ form {
+    @include _margin(0px, 10px, 5px, 25px);
   }
 
 }
 
-.progress-arrow__back {
-  @include _margin(0px, -3px, 0px, 0px);
+.onboarding__step__text {
+  align-items: center;
+  display: inline-flex;
+  justify-items: center;
 }
 
-.progress-arrow__body {
-  @include _margin(0px, -3px, 0px, 0px);
-  @include _padding(5px, 10px, 17px, 15px);
-  height: 50px;
-  text-align: left;
-  width: 215px;
-
-  * {
-    display: inline-block;
-  }
+.onboarding__step--done {
+  background: $color-onboarding-step-done-background;
+  border-bottom: 1px solid $color-background-secondary;
+  border-top: 1px solid $color-background-secondary;
+  color: $color-onboarding-step-done-text;
+}
 
-  p {
-    @include _padding(0px, 0px, 0px, 10px);
-  }
+.onboarding__step--done:before {
+  background: transparent;
+  border-right: 1px solid $color-background-secondary;
+  border-top: 1px solid $color-background-secondary;
+  content: "";
+  height: 35px;
+  left: 0;
+  position: absolute;
+  top: 0;
+  transform: rotate(45deg);
+  transform-origin: 0 0;
+  width: 35px;
+}
 
+.onboarding__step--done:after {
+  background: transparent;
+  border-right: 1px solid $color-background-secondary;
+  border-top: 1px solid $color-background-secondary;
+  content: "";
+  height: 35px;
+  position: absolute;
+  right: -35px;
+  top: 0;
+  transform-origin: 0 0;
+  transform: rotate(45deg);
+  width: 35px;
 }
 
-.progress-arrow__body--undone {
-  background-color: $color-background-tertiary;
-  border-bottom: 1px solid $color-background-tertiary;
-  border-top: 1px solid $color-background-tertiary;
+.onboarding__step--in-progress {
+  background: $color-onboarding-step-in-progress-background;
 
   p {
-    color: $color-text-secondary;
+    color: $color-onboarding-step-in-progress-text;
   }
 
 }
 
-.progress-arrow__body--in-progress {
-  background-color: $color-background-secondary;
-  border-bottom: 1px solid $color-background-secondary;
-  border-top: 1px solid $color-background-secondary;
+.onboarding__step--in-progress:before {
+  border-bottom: 25px solid transparent;
+  border-left: 25px solid $color-background-teach;
+  border-top: 25px solid transparent;
+  content: "";
+  left: 0;
+  position: absolute;
+  top: 0;
+}
 
-  p {
-    color: $color-text-tertiary;
-  }
+.onboarding__step--in-progress:after {
+  border-bottom: 25px solid transparent;
+  border-left: 25px solid $color-onboarding-step-in-progress-background;
+  border-top: 25px solid transparent;
+  content: "";
+  position: absolute;
+  right: -25px;
+  top: 0;
 }
 
-.progress-arrow__body--done {
-  background-color: $color-background-teach;
-  border-bottom: 1px solid $color-background-secondary;
-  border-top: 1px solid $color-background-secondary;
+.onboarding__step--todo {
+  background: $color-onboarding-step-todo-background;
+  color: $color-onboarding-step-todo-text;
+}
 
-  p {
-    color: $color-text-secondary;
-  }
+.onboarding__step--todo:before {
+  border-bottom: 25px solid transparent;
```

Comment:
And more.  It looks like you could use an `onboarding__step:before` definition before you start mucking about with the `todo` and `done` modifiers.